form/form.go

```diff
@@ -542,6 +543,7 @@ func (f *Values) Add(key, val string) {
 // ("bar=baz&foo=quux").
 func (f *Values) Encode() string {
 	var buf bytes.Buffer
+	sort.Sort(f.values)
```

Comment:
I feel that having everyone pay the cost of sorting in production is a bad tradeoff to enable a particular testing scenario.

Reply:
With regard to stripe API latency, I guess sorting a few keys, where most of them are already in the correct order, is of least concern for most users :sweat_smile: 
But your call, I guess the only way to make this a configurable feature will be to implement a custom backend that does deterministic encoding.

Reply:
To clarify: most keys will be in order already because most of the time structs are encoded. As these are auto generated with sorted keys already, and reflection walks the struct keys in the order they are specified, this will not affect most fields.

Reply:
> most keys will be in order already

they'll be in order of addition but not sorted. A lot of the time the entire set will get resorted.